assets/src/scripts/d3-rendering/info-tooltip.js

```diff
@@ -18,5 +18,5 @@ export const appendInfoTooltip = function(elem, text) {
         .attr('class', 'fas fa-info-circle');
 
     // Need to initialize USWDS tooltip explictly after page load
-    components.tooltip.init(elem.node());
+    components.tooltip.on(elem.node());
```

Comment:
USWDS just updated their guidance to include a description on how to dynamically initialize their components and recommend that the "on" function is used.